src/locales/en/mystery-encounter.ts

```diff
@@ -82,4 +82,25 @@ export const mysteryEncounter: SimpleTranslationEntries = {
   "fight_or_flight_option_3_selected": "You leave the strong Pokémon\nwith its prize and continue on.",
   // "fight_or_flight_outro_win": "The mysterious challengers were defeated!",
 
+  "sleeping_snorlax_intro_message": `As you walk down a narrow pathway, you see a towering silhouette blocking your path.
+  $You get closer to see a Snorlax sleeping peacefully.\nIt seems like there's no way around it.`,
+  "sleeping_snorlax_title": "Sleeping Snorlax",
+  "sleeping_snorlax_description": "Do you attack it to try and get it to move, or wait for it to wake up?",
+  "sleeping_snorlax_query": "What will you do?",
+  "sleeping_snorlax_option_1_label": "Fight it",
+  "sleeping_snorlax_option_1_tooltip": "(+) Fight Sleeping Snorlax",
+  "sleeping_snorlax_option_2_label": "Wait for it to move",
+  "sleeping_snorlax_option_2_tooltip": "(75%) Party member falls asleep\n(25%) All Pokémon healed",
```

Comment:
My design philosophy around odds is to not show them so transparently, and leave room for player exploration. In fact I'd just a drop a big "(?)" here and let players wonder what's gonna happen entirely. Or some generic description like "Better safe than sorry".

Reply:
We come from different schools of thought here. Though I understand the appeal of letting players explore, I think the static nature of these encounters means that we should be transparent with them about their odds.
I believe that showing percentages - as well as showing explicit outcomes - leads to a better gameplay experience. 

This is because it enables players to make meaningful decisions about their runs. If they're going to gamble, they should know what the odds are and what the stakes are. Otherwise, you'll end up with players who roll a bad outcome once or twice, and builds up an instinct never to trust the encounter; it effectively robs them of choice because they won't feel like they have any other options. 

If we hide information, what we're really doing is incentivizing people to look up the info in a wiki every time they come across a new event. If we give them the information, they can make a meaningful choice here and now,

Reply:
Ok so I get your point. Let's look at the genre as a whole to see what seems most predominant.

Roguelikes that I know that do show you the odds: Monster Train.

Roguelikes that I know that do not show you the odds: Hades 1 & 2, StS? (not sure about this one), Breach Wanderers, FTL.

@ben-lear can drop on this and break the stalemate, but exploration is important to me. Once they have explore or if they want to check the wiki, they know the odds, so they get the best of both worlds at the end of the day. It's like layers, you have the fresh experience and the seasoned experience.

PS: I think the very unlucky ones will just say "aaaaarfhhf" in the Discord and then be told the odds and that they were just unlucky, much like it happens with eggs. In this same vein, we also don't have the exact odds of the gachas written in game, I don't think they need to be either, much like this.

Reply:
Happy to let @ben-lear be the tiebreaker.
My philosophy actually comes out of an older classic roguelike called Brogue.
<img width="1209" alt="Screen Shot 2024-06-23 at 6 31 32 PM" src="https://github.com/InnocentGameDev/PokeRogue-Events/assets/1934404/f3fa602a-44e2-4b80-978b-2b59229621ae">
I loved the way it showed this information, because it felt like I could make informed decisions as a player that really made each run feel like my own. Note that this was in combat, though, not like encounters like we're talking about here. 

Slay the Spire does do % on encounters, and I think this one is a great example:
![](https://miro.medium.com/v2/resize:fit:4800/format:webp/1*xznF9Ie9jpnTx4Ddfxu8UA.png)

Reply:
Thanks for hearing me out, appreciate it @InnocentGameDev

Reply:
(Never played Brogue but I know of it as I tried to make a copycat of it in Python, following a tutorial!)

It's true that PKMN doesn't like showing you the numbers, but even if that's a thing that could "bolster" my argument, I don't think that should be considered. The two games have a bit of different audiences, I think roguelike players do want things told to them like they are adults -- which they are for the most part. I actually like Smogon's descriptions on moves better.

For Hades, I was mostly thinking of the random boon bags you buy at Charon's, or Boons from Poseidon that give you a random amount of treasure, but honestly it doesn't quite fit the discussion here. 

The screenshot you provided from StS is great. Monster Train has this Event as well. But I don't think it's full clarity either. Here's my example:

![zasds1qgktr61-3158191277](https://github.com/InnocentGameDev/PokeRogue-Events/assets/170796740/6daf0d49-485e-40e6-bd0a-54d5f4ce3be3)

The Event tells you you get a relic, and it could very well tell you you always get N'loth's Gift, but it doesn't. Same with the faces, it might tell you the 50/50, but it doesn't tell you which masks are there -- you either find out through experience (exploring) or wiki it.

The way FTL/Breach Wanderes does it probably the closest approach to what I'd like to see implemented. If you get a chance, try out FTL and see if you like that approach.

I'm not entirely convinced, but I think I'd be a fool if I said you didn't present good examples of how other game devs went about it. And StS is I believe the most popular in this genre. I think arguments can be made for both approaches, so this ain't the hill I'll die on. We can do full transparency if that's something you guys would like.

Here's a middle ground approach I propose:

"(75%) Rest" -- written in red.
"(25%) Rest" -- written in green.

This would show the numbers and the chances of a good/bad outcome, while not taking away the wondering of "wait why is one rest good and the other bad? guess only one way to find out" (and that "one way" they're thinking of isn't heading straight to the wiki, I assume)